src/features/import-map.js

```diff
@@ -11,15 +11,16 @@
  * There is no support for dynamic import maps injection currently.
  */
 import { baseUrl, resolveAndComposeImportMap, resolveImportMap, resolveIfNotPlainOrUrl, hasDocument } from '../common.js';
+import { errMsg } from '../err-msg.js';
 import { systemJSPrototype } from '../system-core.js';
 
 let importMap = { imports: {}, scopes: {} }, importMapPromise;
 
 if (hasDocument) {
   iterateImportMaps(function (script) {
     script._j = fetch(script.src).then(function (res) {
-      return res.json();
-    });
+      return res.text();
+    }).then(parseJson);
```

Comment:
I just was looking at this in the other PR actually - the `parseJson`'s can be combined by making it a `script._t` with the change below.